usr/src/test/os-tests/tests/pf_key/acquire-compare.sh

```diff
@@ -37,6 +41,12 @@ fi
 
 # NOTE: If multihomed, this may fail in interesting ways...
 MY_IP=`netstat -in -f inet | egrep -v "Name|lo0" | awk '{print $4}' | head -1`
+
```

Comment:
A further reality check might be to check that TEST_REMOTE_DST* (see below) hit a route, and one that isn't RTF_BLACKHOLE (which returns ENETUNREACH).  Even if they packets go off to hyperspace, the netstack must think they can leave out the {,v}NIC successfully.

Reply:
I'm not sure how to easily achieve that, we can't use `ping` since they don't have to _really_ be reachable, based on what you said.  Is there an easy way I'm not network-savvy enough to know?  I'm willing to leave it in the state it's in, for right now, since it's better than before.

Reply:
route(8) is your friend here but it's... potentially annoying in corner cases.  THere are other failures that might cut off the protect-with-IPsec path too.  And also, as I think about it, there might be lack of "-n" (no name resolution) in some commands.  I'll have to look at it more deeply later when I have cycles.